tools/mo/openvino/tools/mo/graph/port.py

```diff
@@ -274,6 +274,29 @@ def get_tensor_names(self, port_renumber: bool = False):
                     tensor_names_list.append(tensor_name.replace(',', '\\,'))
         return sorted(tensor_names_list)
 
+    def add_tensor_names(self, op_names: list, tensor_names: list, port_renumber: bool = False):
```

Comment:
not clear why operation name is needed. Please clarify

Reply:
Removed op_names from add_tensor_names() method and used op name from port.